internal/api/handlers_test.go

```diff
@@ -336,8 +344,15 @@ func TestGoCertRouter(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			if res.StatusCode != tC.status || string(resBody) != tC.response {
-				t.Errorf("expected response did not match.\nExpected vs Received status code: %d vs %d\nExpected vs Received body: \n%s\nvs\n%s\n", tC.status, res.StatusCode, tC.response, string(resBody))
+			switch path := tC.path; path {
+			case "/metrics":
+				if res.StatusCode != tC.status || !strings.Contains(string(resBody), tC.response) {
+					t.Errorf("expected response did not match.\nExpected vs Received status code: %d vs %d\nExpected vs Received body: \n%s\nvs\n%s\n", tC.status, res.StatusCode, tC.response, string(resBody))
+				}
+			default:
+				if res.StatusCode != tC.status || string(resBody) != tC.response {
+					t.Errorf("expected response did not match.\nExpected vs Received status code: %d vs %d\nExpected vs Received body: \n%s\nvs\n%s\n", tC.status, res.StatusCode, tC.response, string(resBody))
```

Comment:
You can remove the switch statement and just use `strings.Contains` for either case, since it will return true if the strings match precisely too.

Reply:
Done, thanks